iceberg/bodo_iceberg_connector/iceberg-java/src/main/java/com/bodo/iceberg/BodoIcebergHandler.java

```diff
@@ -59,19 +57,6 @@ private Table loadTable(String tableId) {
     return catalog.loadTable(TableIdentifier.parse(tableId));
   }
 
-  /**
```

Comment:
These are all functions with features superseded by PyIceberg. Since these are all table metadata operations, it doesn't really make sense to have a Java equivalent because they are all lightweight operations as is. So just using PyIceberg for these